src/ui/component/collapsible/collapsible.scss

```diff
@@ -32,14 +32,14 @@
 }
 
 .okp4-dataverse-portal-collapsible-content[data-state='open'] {
-  animation: slideDown 300ms ease-out;
+  animation: collapsible-expand 300ms ease-out;
 }
 
 .okp4-dataverse-portal-collapsible-content[data-state='closed'] {
-  animation: slideUp 300ms ease-out;
+  animation: collapsible-collapse 300ms ease-out;
 }
 
-@keyframes slideDown {
+@keyframes collapsible-expand {
```

Comment:
Since keyframes names are globals, it might be better to keep the prefix to avoid collisions. The `expand` keyframe is currently not generic because is uses the custom property related to radix element's size.
We can either:
- make it generic (e. g. by using a mixin)
- keep it specific, with this naming or renaming it differently depending on what we want as prefix (the file name was chosen here).

What do you think?

Reply:
You're right, si it's better to keep it named as you did